vhost-device-scmi/CHANGELOG.md

```diff
@@ -9,16 +9,16 @@
 
 ### Deprecated
 
-## [0.2.0]
+## v0.2.0
```

Comment:
What do you think about adopting a specific CHANGELOG format like https://common-changelog.org/ ? There seem to be some tools that verify the markdown doc is compliant to specific CHANGELOG formats, but I haven't checked them out myself yet.

Reply:
I like the idea, in this case I tried to follow https://github.com/rust-vmm/community/blob/main/docs/crate_release.md , so should we update also that ?

I think it is not strict, but having a common thing and maybe integrating the checks into the CI would not be bad.

Reply:
Yeah makes sense! I made an issue here: https://github.com/rust-vmm/community/issues/182